extras/wificfg/wificfg.c

```diff
@@ -1856,8 +1894,10 @@ void wificfg_init(uint32_t port, const wificfg_dispatch *dispatch)
         wifi_mode = STATIONAP_MODE;
     else if (wifi_sta_enable)
         wifi_mode = STATION_MODE;
-    else
+    else if (wifi_ap_enable)
         wifi_mode = SOFTAP_MODE;
+    else
+        printf("Warning: No AP/STA enabled by wificfg.\n");
```

Comment:
It is quite normal for the wifi to not be running, so I would not bother with this. There is already enough output to see which interfaces have been started.